crates/y-sweet-core/Cargo.toml

```diff
@@ -10,21 +10,21 @@ repository = "https://github.com/drifting-in-space/y-sweet"
 [dependencies]
 anyhow = "1.0.72"
 async-trait = "0.1.71"
-base64 = "0.21.2"
 bincode = "1.3.3"
+data-encoding = "2.4.0"
 getrandom = { version = "0.2.10", features = ["js"] }
 lib0 = "0.16.9"
 rand = "0.8.5"
+reqwest = "0.11.18"
```

Comment:
`reqwest` and `time` are merely moved into the sorted order here, the imports are the same.